lib/repo/subversion_repository.rb

```diff
@@ -487,7 +493,13 @@ def self.delete_bulk_permissions(repo_names, user_ids)
       if !Repository.conf[:IS_REPOSITORY_ADMIN] # Are we admin?
         raise NotAuthorityError.new("Unable to delete bulk permissions:  Not in authoritative mode!");
       end
-
+      # Check if configuration is in order
+      if Repository.conf[:REPOSITORY_PERMISSION_FILE].nil?
+        raise ConfigurationError.new("Required config 'REPOSITORY_PERMISSION_FILE' not set")
+      end
+      if !File.exist?(Repository.conf[:REPOSITORY_PERMISSION_FILE])
+        File.open(Repository.conf[:REPOSITORY_PERMISSION_FILE], "w").close() # create file if not existent
```

Comment:
Line is too long. [106/80]<br>Do not use parentheses for method calls with no arguments.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.